ui-manchette-with-spacetimechart/src/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -54,12 +62,6 @@ const useManchettesWithSpaceTimeChart = (
 
   const paths = usePaths(projectPathTrainResult, selectedTrain);
 
-  // Memoize timeWindow to avoid recalculation on each render
-  const timeWindow = useMemo(
-    () => computeTimeWindow(projectPathTrainResult),
```

Comment:
Nit: this was the only caller of `computeTimeWindow()`, so I think this function can be dropped?